Dockerfile-api

```diff
@@ -0,0 +1,13 @@
+FROM node:boron
+# Create app directory and define work dir for api
+WORKDIR /usr/src/dc-campaign-finance-watch/api
+
+# Install app dependencies
+COPY package.json .
+RUN npm install
+
+# Copy the contents of current directory to work dir
+COPY . .
+
+# Wait for redis to be up then start api
+CMD ["./scripts/wait-for-it.sh","redis:6379","--","npm", "run", "dev"]
```

Comment:
Similar comment, I think you can use ENTRYPOINT for the wait-for-it.sh, and then use CMD to pass the values to it.